src/middlewares/user.middleware.js

```diff
@@ -146,3 +146,17 @@ export const removeAuthCode = async(req, res, next)=>{
   await UserService.updateUser({authCode}, req.user.id);
   next();
 };
+
+export const resetToken = async(req,res,next) =>{
+	const {token} = req.params
+      const ver = JwtUtility.verifyToken(token)
+      const id = ver.user_id
+	  req.id = id
+      const user = await User.findOne({
+        where: { id: id },
+      });
+      if (!user) {
+      return res.status(404).json({ message: `user does not exist` });
+      }
+	  next();
+}
```

Comment:
- Let use `findByPk` instead of `findOne`
- You need to have a meaningful variable. What is ver?
- Rename your middleware to `VerifyResetPasswordToken`